README.md

```diff
@@ -34,23 +53,122 @@ They are said to be "bound" to the unit they are currently operating on.
+even if it had more instructions left to execute, same as in mvr. 
+
+#### cop
+
+Places the complement of the current base into the complementary strand, and enters "copy mode." Each base has its corresponding
+complement, known as the complementary base pairin (see [Definitions](#Definitions)). Copy mode remains in effect until an off 
+instruction is reached (or the end of the strand/enzyme). While in copy mode, any base that is "touched" by the enzyme will 
+have its complementary base placed above it in the complementary strand. When bases are copied into the complementary 
+strand, they are placed *upside-down* above the element copied, so that the strands are mirrored to each other. Copy mode applies 
+after a move, so that the movement off of a strand or into a gap between bases still preserves the behavior of the strand ending normally.  
+While in copy mode, additional cop instructions do nothing. 
+
+#### off
+
+Turns off copy mode. If copy mode is not enabled, then this does nothing. 
+
+#### ina
+
+Inserts an 'A' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (T) is placed above into
+the complementary strand. 
+
+#### inc
+
+Inserts an 'C' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (G) is placed above into
+the complementary strand. 
+
+#### int
+
+Inserts an 'T' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (A) is placed above into
+the complementary strand. 
+
+#### ing
+
+Inserts an 'G' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (C) is placed above into
+the complementary strand. 
+
+#### rpy
+
+Moves to the nearest pyrine to the right of this unit. (See [Definitions](#Definitions)). With copy mode enable, each base touched while sliding to the next pyrine 
+will be copied into the complementary strand. If there are no pyrines to the right of the current base, this does nothing.
+
+#### rpu
+
+Moves to the nearest purine to the right of this unit. (See [Definitions](#Definitions)). With copy mode enable, each base touched while sliding to the next purine 
+will be copied into the complementary strand. If there are no purines to the right of the current base, this does nothing.
+
+#### lpy
+
+Moves to the nearest pyrine to the left of this unit. (See [Definitions](#Definitions)). With copy mode enable, each base touched while sliding to the next pyrine 
+will be copied into the complementary strand. If there are no pyrines to the left of the current base, this does nothing.
+
+#### lpu
+
+Moves to the nearest purine to the left of this unit. (See [Definitions](#Definitions)). With copy mode enable, each base touched while sliding to the next purine 
+will be copied into the complementary strand. If there are no purines to the left of the current base, this does nothing.
```

Comment:
Maybe move the `(See [Definitions](#Definitions))` bit to be directly after the pyrine/purine word?

Reply:
Also it should probably be `With copy mode enabled,...` instead of `With copy mode enable,...`.

Reply:
In the case that there is no purines/pyrines to the left/right, does it actually do nothing? Or does it try to go that direction until it falls off? Because if copy mode is on then it is filling in all those empty complements till the end of the strand.

Reply:
> In the case that there is no purines/pyrines to the left/right, does it actually do nothing? Or does it try to go that direction until it falls off? Because if copy mode is on then it is filling in all those empty complements till the end of the strand.

Another ambiguous point, so we should make an implementation decision. Secondarily, we have to decide if these instructions are capable of traversing gaps in strands. 
I would argue that the behavior is more interesting if they do nothing if there are no left/right purines/pyrines, as then there will be more strands that do not terminate prematurely in the space of all possible strands. I also think that they should not traverse gaps on the strand that they act on, as regular moves cannot traverse gaps either.

Reply:
Talked with Finn. We're going to make it so that these instructions dont have a look ahead behavior.